### PR TITLE
106 authorization

### LINKED ISSUE #106: Authorization: Expose email address of owner
**Describe the bug**
Given I am authenticated, I should be authorized to request my own email address. 
**To Reproduce**
When I send the following to the backend:
```graphql
{
   User(email: "user@example.org") {
    email
    name
  }
}
```
I expect to see:
```json
{
  "data": {
    "User": [
      {
        "email": "user@example.org",
        "name": "Jenny Rostock"
      }
    ]
  }
}
```
But instead I see:
```json
{
  "data": {
    "User": [
      {
        "email": null,
        "name": "Jenny Rostock"
      }
    ]
  },
  "errors": [
    {
      "message": "Not Authorised!",
      "locations": [
        {
          "line": 3,
          "column": 5
        }
      ],
      "path": [
        "User",
        0,
        "email"
      ]
    }
  ]
}
```

**Expected behavior**
A clear and concise description of what you expected to happen

**Additional context**

### PR DESCRIPTION
**Pull request by roschaefer**
Fix #106

### CODE REVIEW

**Comment by roschaefer** [comment]
@appinteractive can we remove the stupid WIP bot?

**Comment by roschaefer** [comment]
@appinteractive I just removed the WIP bot, feel free to install it if you think we need it.

**Comment by roschaefer** [comment]
@appinteractive can be merged

File: src/middleware/passwordMiddleware.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ export default {
   Query: async (resolve, root, args, context, info) => {
     const result = await resolve(root, args, context, info)
     return walkRecursive(result, ['password'], () => {
-      // replace password with asterix
+      // replace password with asterisk
```
Comment (appinteractive) [inline src/middleware/passwordMiddleware.js]: Yeah Obelix could have done it too 😬

**Pull request merged**